### ISSUE TITLE
Docs contributing

### ISSUE THREAD

**Issue by chrisarcand** [phoenixframework/phoenix_guides#663]
I'm new to both Elixir and using Phoenix, so bear with me if I'm missing a lot of common knowledge with running Elixir applications.

The first thing I do when I try out a new thing is make sure to contribute to the documentation, because having 'fresh eyes' as a newcomer to a project is something you only get to truly experience once. However, I've found contributing to these guides fairly frustrating.

* There's no instructions in the README about running/compiling the documentation to view locally at all. 
* I did some digging and determined I can generate docs with `mix deps.get` and then `mix docs` to generate documentation with ex_doc. That's sort of ok, but the ex_doc really doesn't follow the site very well at all, with different sections and styling, etc.

Take for example...

![](http://screenshots.chrisarcand.com/r3ysw.jpg)

vs.

![](http://screenshots.chrisarcand.com/kwe2l.jpg)

I found the desciption in the mix.exs saying that "The guides are published from the phoenixframework/phoenix project, not separately, however this config exists to make it easier to preview changes to the guides without also building the framework source API docs." [[ref](https://github.com/phoenixframework/phoenix_guides/commit/6ac0f7c2cc05bf8cab9f9dd6cb5c534f22693677)] , so would expect that the markdown here doesn't display quite correctly, ~~but I don't know where to go from there.~~ Given the differences, maybe contributors should just be directed to run docs in the main phoenix repo instead, really.

* I did more digging and found the styling at https://github.com/phoenixframework/phoenix_guides/tree/master/styling But then I notice that this is actually the styling from 0.10.0 and doesn't at reflect what's on phoenixframework.org. 

The original reason I came here was to try and fix this:

![](http://screenshots.chrisarcand.com/c19ri.jpg)

Since starting to look at Phoenix, it's driven me crazy that every time I read something and note it's from a beta version, when I go to the top and try to select the newest I have to "View all versions" because the very first betas are listed first. Is this just something you have to deal with because of readme.io...?

---

That's a lot of info I know, but I'm trying to get set up to help out; there are a few weirdnesses I thought I'd ask about first, and the guides don't appear to have been touched in months. 😕

**Comment by EduardoBautista** [comment]
I think the best approach here would be to just fork this and take over the guides.

**Comment by willykaram** [comment]
@chrisarcand thanks for the feedback and sorry to hear that you're finding contributing to the guide a bit frustrating. I made some small contributions over the last year or two. But unfortunately, I wasn't able to contribute much since then as things at work ramped up.

That said, I'm going to be resuming my contribution efforts on the guides. I'm going to be making a  big push to help @Gazler with the guides in relation to the Phoenix 1.3 release and beyond (@chrismccord mentioned he's taken up much of the effort while @lancehalvorsen was working on this new book, which I recommend reading: https://pragprog.com/book/lhelph/functional-web-development-with-elixir-otp-and-phoenix ).

So I'll do what I can to assist the issues you raised above, as we want to make it as easy as possible for people to contribute to the docs. I'd ideally also like to see us build a distributed team of folks interested in helping with the guides, so if you're up for chipping in, please LMK.

**Comment by rakr** [comment]
Beginner with both Elixir and Phoenix, I'm trying to document my learnings as much as possible, happy to create contribute on a "Phoenix for Dummies" kind of section

**Comment by chrisarcand** [comment]
Closing this in favor of adding it to the items included in https://github.com/phoenixframework/phoenix_guides/issues/680. Thanks @chrismccord

**Comment by lizvdk** [comment]
As a (hopefully) new contributor, I suggest that details about the 2-step review process mentioned by @willykaram [here](https://github.com/phoenixframework/phoenix_guides/issues/680#issuecomment-301762082) be added to [CONTRIBUTING.md](https://github.com/phoenixframework/phoenix_guides/blob/master/CONTRIBUTING.md).

**Comment by willykaram** [comment]
@lizvdk thanks, that a good suggestion. Although can we hold off on making that change to the guide or at least submitting the PR? I'm also a relatively new contributor/not a member of the Phoenix team (I've only created few guides/edits so far), so I was was thinking it would just be helpful to do some peer review amongst ourselves before submitting a PR to the Phoenix team, so as to be efficient with their time (that's an approach that's worked well on docs for some other open source projects I've worked on). I was thinking that if we and the Phoenix team find that process is working well here, then we'd update the guide. Although, of course, I'll defer to @Gazler and @jeregrine on that if they think otherwise.
